js/bibox.js

```diff
@@ -65,35 +65,71 @@ module.exports = class bibox extends Exchange {
                 'www': 'https://www.bibox365.com',
                 'doc': [
                     'https://biboxcom.github.io/en/',
+                    'https://biboxcom.github.io/v3/spot/en/',
                 ],
                 'fees': 'https://bibox.zendesk.com/hc/en-us/articles/360002336133',
                 'referral': 'https://w2.bibox365.com/login/register?invite_code=05Kj3I',
             },
             'api': {
                 'public': {
-                    'post': [
-                        // TODO: rework for full endpoint/cmd paths here
-                        'mdata',
-                    ],
                     'get': [
-                        'cquery',
-                        'mdata',
-                        'cdata',
-                        'orderpending',
+                        'mdata/ping', // v3
```

Comment:
I would suggest a slightly different api structure here:

```JavaScript
'api': {
    'v1': {
        'public': {
            'get': {
                'mdata': 1, // this is the rate limiter cost, need to be aligned with their docs
                ...               
            },
            ...
        },
    },
    'v3': {
        'public': {
            'get': {
                'mdata/ping': 1, // this is the rate limiter cost, need to be aligned with their docs
                ...               
            },
            ...
        },
    }
}
```

If we use this, the 'api' arg of `sign()` will be an array of paths to the endpoint, see how it's done in these files:

<img width="431" alt="Screenshot 2022-05-10 at 03 14 14" src="https://user-images.githubusercontent.com/1294454/167518309-ab726911-e9e5-4110-b6c8-1b837be2da0f.png">